Sources/RTVIClientIOS/RTVIClient.swift

```diff
@@ -6,7 +6,7 @@ open class RTVIClient {
     
     private let options: RTVIClientOptions
     private var transport: Transport
-    private var baseUrl: String
+    private var baseUrl: String? // see RTVIClientParams for explanation of optionality
```

Comment:
If I’m not mistaken, we have already removed this option from our JavaScript SDK.

Since you are already making changes, would it be worthwhile to remove it here as well now? Or should we include its removal as [part of this task](https://linear.app/dailyco/issue/BOT-274/improve-ios-sdk), along with a couple of other small refactors needed in our iOS SDK?

Reply:
Ah I see. Looks like in the JS SDK `baseUrl` now lives nested under `params` only and not at the "top level", is that right?

I wasn't necessarily looking to make any breaking changes with this PR; maybe we could save this cleanup for the next batch of work?